nominal/nominal.py

```diff
@@ -425,27 +434,37 @@ def get_asset(rid: str) -> Asset:
     return conn.get_asset(rid)
 
 
+@deprecate_keyword_argument("properties", "property")
+@deprecate_keyword_argument("labels", "label")
 def search_assets(
     *,
     search_text: str | None = None,
     label: str | None = None,
+    labels: Sequence[str] | None = None,
     property: tuple[str, str] | None = None,
-) -> list[Asset]:
+    properties: Mapping[str, str] | None = None,
+) -> Sequence[Asset]:
     """Search for assets meeting the specified filters.
+    Filters are ANDed together, e.g. `(asset.label == label) AND (asset.search_text =~ field)`
 
-    Filters are ANDed together, e.g. `(asset.label == label) AND (asset.property == property)`
-    - `search_text`: search case-insensitive for any of the keywords in all string fields.
-    - `property` is a key-value pair, e.g. ("name", "value")
+    Args:
+        search_text: case-insensitive search for any of the keywords in all string fields
+        label: Deprecated, use labels instead.
```

Comment:
Necessary to keep the old arg in the docstring? Old code will warn, new code should not need it. (Perhaps for people reading old code?) I see some tools add a [special section for deprecations and their versions](https://mkdocstrings.github.io/griffe/reference/docstrings/#google-section-deprecated).